pingcap/tidb-operator
# update lightning&br version to v4.0.2

### What problem does this PR solve? 
Resolve https://github.com/pingcap/tidb-operator/issues/2817.
Current lightning doesn't support auto random.

### What is changed and how does it work?
1. Update lightning&br version to v4.0.2 to support auto random.
2. Update `local-up-operator.sh` to support using local backup image.

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)

test with different tidb cluster versions to check whether br/lightning support `auto random`. Tested versions are v3.1.0, v3.1.2, v4.0.0, v4.0.1

For lightning, all versions can import data both with auto_random and without auto_random successfully.
For br, br can't restore v3.1.0 cluster(I think this is a known issue). The other versions are ok.

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
Update the lightning used in tidb-backup-manager from v3.0.15 to v4.0.2
```
